pages/dashboard/projects/client/invoice/[id].js

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect, useMemo } from "react";
+import styled from "styled-components";
+import Nav from '../../../../../components/unzipped/header';
+import Invoice from "../../../../../components/unzipped/dashboard/project/invoice";
+import { useRouter } from 'next/router';
+import { getBusinessTasksByFounder } from "../../../../../redux/actions";
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux'
+
+const Desktop = styled.div`
+@media(max-width: 680px) {
+    display: none;
+}
+`
+const MobileDisplayBox = styled.div`
+    @media(min-width: 680px) {
+        display: none;
+    }
+`;
+
+const FounderInvoice = ({ projectName, invoiceTaskHours, access_token, getBusinessTasksByFounder }) => {
+    const router = useRouter();
+    const { id } = router.query;
+
+    const [data, setData] = useState([]);
+    const [filteredData, setFilteredData] = useState([]);
+    const [sortedData, setSortedData] = useState({})
+    const [selectedWeek, setSelectedWeek] = useState(null);
+    const [weekOptions, setWeekOptions] = useState([]);
+    const [take, setTake] = useState(25);
```

Comment:
is there any option for user to select number of invoices to be shown on gird like a dropdown 10,15,20,25,50,100 like that or is it fixed to 25?

Reply:
No, it vary. There are three options on top 25 50 100.